packages/yoroi-extension/app/UI/features/portfolio/useCases/TokenDetails/HeaderDetails/Header.tsx

```diff
@@ -48,21 +48,29 @@ const HeaderSection = observer(({ tokenInfo, stores }: Props): React.ReactNode =
     tokenActivity: { data24h },
   } = usePortfolioTokenActivity();
 
+  const getClosePrice = (): number | null => {
+    if (data24h && data24h[tokenInfo.info.id] && data24h && data24h[tokenInfo.info.id].length > 1) {
```

Comment:
Are we sure that `tokenInfo` will have `info.id`. Maybe good to do null checks"


```suggestion
    if(tokenInfo?.info?.id){
    if (data24h && data24h[tokenInfo.info.id] && data24h && data24h[tokenInfo.info.id].length > 1) {
    ...
    }
 }
```